src/main/java/ch/njol/skript/conditions/CondIsSkriptCommand.java

```diff
@@ -18,38 +18,52 @@
  */
 package ch.njol.skript.conditions;
 
-import static ch.njol.skript.command.Commands.skriptCommandExists;
-
-import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.Skript;
+import ch.njol.skript.command.Commands;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Is a Skript command")
 @Description("Checks whether a command/string is a custom Skript command.")
-@Examples(
-	{"# Example 1",
+@Examples({
+	"# Example 1",
 	"on command:", 
 	"\tcommand is a skript command",
 	"",
 	"# Example 2",
 	"\"sometext\" is a skript command"})
-@Since("2.6")
-public class CondIsSkriptCommand extends PropertyCondition<String> {
+@Since("2.6, INSERT VERSION (aliases support)")
+public class CondIsSkriptCommand extends Condition {
```

Comment:
Actually not sure why I did, if anything it's because I wanted to prefix it with `command[s]` as it seems right. Since from what I understand there is no way to add a prefix to it